bot/event_handler.py

```diff
@@ -65,7 +69,10 @@ def _handle_message(self, event):
 
         # Remove mention of the bot so that the rest of the code doesn't need to
         msg_txt = self.clients.remove_mention(msg_txt).strip()
-        # Possible error if hal is mentioned in message to hal?
+
+        # Ensure that we don't go to wit with messages posted by slackbot
+        if event['user'] == "USLACKBOT":
```

Comment:
Good idea. I've updated it in my repo.

John

On Mon, Jun 27, 2016 at 9:47 AM, GalateaBerto <notifications@github.com>
wrote:

> In bot/event_handler.py
> <https://github.com/GalateaRaj/galatea-hal/pull/2#discussion_r68578579>:
>
> > @@ -65,7 +69,10 @@ def _handle_message(self, event):
> >
> >          # Remove mention of the bot so that the rest of the code doesn't need to
> >          msg_txt = self.clients.remove_mention(msg_txt).strip()
> > -        # Possible error if hal is mentioned in message to hal?
> > +
> > +        # Ensure that we don't go to wit with messages posted by slackbot
> > +        if event['user'] == "USLACKBOT":
>
> Would be good to have a set of ignored users and add this user ID to that
> set. I assume there will be other bots we'll want to ignore in the future.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/GalateaRaj/galatea-hal/pull/2/files/bfb098311d28fc4365e24b3c636c5583ed99bb1b#r68578579>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ATKQPmSbeqo8z4BQQRFYSQGYINpa3XX_ks5qP9RygaJpZM4I_Eyh>
> .
>